mealy/metrics.py

```diff
@@ -35,54 +35,58 @@ def fidelity_balanced_accuracy_score(y_true, y_pred):
-        report_dict[ErrorAnalyzerConstants.MPP_ACCURACY] = mpp_accuracy_score
-        report_dict[ErrorAnalyzerConstants.MPP_BALANCED_ACCURACY] = mpp_balanced_accuracy
-        report_dict[ErrorAnalyzerConstants.MPP_FIDELITY] = fidelity
+        report_dict[ErrorAnalyzerConstants.EDT_ACCURACY] = edt_accuracy_score
+        report_dict[ErrorAnalyzerConstants.EDT_BALANCED_ACCURACY] = edt_balanced_accuracy
+        report_dict[ErrorAnalyzerConstants.EDT_FIDELITY] = fidelity
         report_dict[ErrorAnalyzerConstants.PRIMARY_MODEL_TRUE_ACCURACY] = primary_model_true_accuracy
         report_dict[ErrorAnalyzerConstants.PRIMARY_MODEL_PREDICTED_ACCURACY] = primary_model_predicted_accuracy
         report_dict[ErrorAnalyzerConstants.CONFIDENCE_DECISION] = confidence_decision
         return report_dict
 
-    report = 'The MPP was trained with accuracy %.2f%% and balanced accuracy %.2f%%.' % (mpp_accuracy_score * 100,
-                                                                                         mpp_balanced_accuracy * 100)
-    report += '\n'
-    report += 'The Decision Tree estimated the primary model''s accuracy to %.2f%%.' % \
-              (primary_model_predicted_accuracy * 100)
-    report += '\n'
-    report += 'The true accuracy of the primary model is %.2f.%%' % (primary_model_true_accuracy * 100)
-    report += '\n'
-    report += 'The Fidelity of the MPP is %.2f%%.' % \
-              (fidelity * 100)
-    report += '\n'
-    if not confidence_decision:
-        report += 'Warning: the built MPP might not be representative of the primary model performances.'
+    elif output_format == 'text':
+
+        report = 'The Error Decision Tree (EDT) was trained with accuracy %.2f%% and balanced accuracy %.2f%%.' % (edt_accuracy_score * 100, edt_balanced_accuracy * 100)
         report += '\n'
-        report += 'The MPP predicted model accuracy is considered too different from the true model accuracy.'
+        report += 'The Decision Tree estimated the primary model''s accuracy to %.2f%%.' % \
+                  (primary_model_predicted_accuracy * 100)
         report += '\n'
-    else:
-        report += 'The MPP is considered representative of the primary model performances.'
+        report += 'The true accuracy of the primary model is %.2f.%%' % (primary_model_true_accuracy * 100)
+        report += '\n'
+        report += 'The Fidelity of the EDT is %.2f%%.' % \
+                  (fidelity * 100)
         report += '\n'
+        if not confidence_decision:
+            report += 'Warning: the built EDT might not be representative of the primary model performances.'
+            report += '\n'
+            report += 'The EDT predicted model accuracy is considered too different from the true model accuracy.'
+            report += '\n'
+        else:
+            report += 'The EDT is considered representative of the primary model performances.'
+            report += '\n'
 
-    return report
+        print(report)
```

Comment:
Done 0f66854. 
I changed it to `print` because it is a bit weird when the function returns a string (user will need to `print` it anw to have a clear paragraph). No strong feeling though.

Reply:
I think it is better as a string. If the user writes the results in a file or whatever.